docs/translating-dev.md

```diff
@@ -21,6 +27,8 @@
 
 ## Things to know/Style Guides
 
-- Do not use it inside ``getDefaultProps`` at the point where ``getDefaultProps`` is initialized the translations aren't loaded yet and it causes missing translations.
-- If using translated strings as constants, translated strings can't be in constants loaded at class-load time since the translations won't be loaded.
+- Do not use `_t()` inside ``getDefaultProps`` at the point where ``getDefaultProps`` is initialized when the translations aren't loaded yet and it causes missing translations. Use `_td()` instead and perform the actual translation later.
```

Comment:
Can I suggest, "Do not use `_t()` inside ``getDefaultProps``: the translations aren't loaded when `getDefaultProps` is called, leading to missing translations. Use `_td` to indicate that `_t` will be called on the string later "